hydra_genetics/utils/units.py

```
@@ -130,7 +131,7 @@ def get_unit_flowcell(units: pandas.DataFrame, wildcards: snakemake.io.Wildcards
     return unit["flowcell"]
 
 
-def get_units(units: pandas.DataFrame, wildcards: snakemake.io.Wildcards, type: str = None) -> pandas.DataFrame:
+def get_units(units: pandas.DataFrame, wildcards: Wildcards, type: str = None) -> pandas.DataFrame:
```

Ruff (0.15.6)

[error] Function argument `type` is shadowing a Python builtin

(A002)